core/domain/collection_services_test.py

```diff
@@ -1844,3 +1844,23 @@ def test_contributor_summary(self):
         #       self.COLLECTION_ID, 4, 3)
         # self._check_contributors_summary(self.COLLECTION_ID,
         #                                 {albert_id: 1, bob_id: 2})
+
+
+class GetCollectionAndCollectionRightsTest(CollectionServicesUnitTests):
+
+    def test_get_collection_and_collection_rights_object(self):
+        collection_id = self.COLLECTION_ID
+        self.save_new_valid_collection(
+            collection_id, self.owner_id, objective='The objective')
+
+        (collection, collection_rights) = (
+            collection_services.get_collection_and_collection_rights_by_id(
+                collection_id))
+        self.assertEqual(collection.id, collection_id)
+        self.assertEqual(collection_rights.id, collection_id)
```

Comment:
Here and above, do you think we should test the other fields of the collection/collection_rights domain objects?

Reply:
I think its fine as we just have to test whether object is fetched or not. If test is required it would be implemented in the controller where it's used.

Also, I have just set collection_id so which other fields might I test anyway.

Reply:
Makes sense! I agree.